project/PACKAGERS.md

```diff
@@ -185,12 +185,6 @@ NOTE: if you need to set more than one build tag, space separate them:
 export DOCKER_BUILDTAGS='apparmor exclude_graphdriver_aufs'
 ```
 
-### LCOW (Linux Containers On Windows)
-
-LCOW is an experimental feature on Windows, and requires the daemon to run with
-experimental features enabled. Use the `no_lcow` build tag to disable the LCOW
-feature at compile time, 
-
```

Comment:
(Off-topic: PACKAGERS.md seems completely outdated 😅)

Reply:
Yes, it needs to be cleaned up